packages/control-hub/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "target": "es2017"
```

Comment:
Is there a reason we need to override this value?

Reply:
This may have been from before the change to es modules in main. It's not needed anymore.